jazz_deployments-event-handler/index.js

```diff
@@ -0,0 +1,162 @@
+/**
```

Comment:
In general i would write the functions like this
export function abc() {
}
rather than var abc = function() and then doing module.exports{abc: abc} as this will not be unit testable.
This is a refactor and you can think about it in separate PR.

Reply:
Also try to use ES6 syntax
arrow functions =>, let keywords, etc

Reply:
not clear

Reply:
lets talk offline about this.